script.js

```diff
@@ -0,0 +1,51 @@
+let api = `https://v6.exchangerate-api.com/v6/${apiKey}/latest/USD`;
```

Comment:
Fiindcă nu schimbi nicăieri valoarea variabilei, o poți face constantă.

```suggestion
const api = `https://v6.exchangerate-api.com/v6/${apiKey}/latest/USD`;
```